src/advanced_unsafety/invalid_values.md

```diff
@@ -90,7 +96,7 @@ As a library user you may not assume anything about the representation of a libr
 ## When you might end up making an invalid value
 
 
-Invalid values have a chance to crop up when you're reinterpreting a chunk of memory as a value of a different type. This can happen when calling [`mem::transmute()`], [`mem::transmute_copy()`], or [`mem::zeroed()`], when casting a reference to a region of memory into one of a different type, or when accessing the wrong variant of a `union`. The value need not be on the stack to be considered invalid: if you gin up an `&bool` that points to a bit pattern that is not a valid `bool`, that is instantly UB even if you don't read from the reference.
+Invalid values have a chance to crop up when you're reinterpreting a chunk of memory as a value of a different type. This can happen when calling [`mem::transmute()`], [`mem::transmute_copy()`], or [`mem::zeroed()`], when casting a reference to a region of memory into one of a different type, or when accessing the wrong variant of a `union`. The value need not be on the stack to be considered invalid: if you gin up an `&bool` that points to a bit pattern that is not a valid `bool`, that can instantly be UB (in a "deep validity" world) even if you don't read from the reference.
```

Comment:
It may be prudent to add `mem::uninitialized` to the list here, unless it is already excluded from consideration elsewhere.

Reply:
We have a separate chapter on uninit and this chapter is explicitly not talking about uninit. But I think I can add a reminder to this section.

Reply:
added